src/modules/presidents/store/mutations.js

```diff
@@ -0,0 +1,10 @@
+const SET_CANDIDATES = (state, candidatesPayload) => {
+  const localState = state;
+  const availableCandidates = candidatesPayload.filter(candidate => candidate.descricaoSituacao === 'Deferido');
+
+  localState.candidates = availableCandidates;
```

Comment:
Por que não usar `Vue.set`?

Reply:
Entao.... importar fazer o import do Vue, utilizar um metodo para dai fazer a atraibuição do objeto. Neste caso especifico não to vendo o beneficio de usar o set.

Reply:
É que nesse caso estás carregando as fotos direto junto com as infos dos candidatos, o que antes estava desacoplado e para fazer alteração no Array seria necessário o uso do Vue.set.

Reply:
Isso !

Eu achava que o `getPhoto` trazia somente a respectiva foto do candidato, mas ele tras todas informações do candidato, então já que vai se fazer a request de todo jeito matamos 2 coelhos numa tacada só.